gnomad_qc/v3/create_release/create_release_sites_ht.py

```diff
@@ -0,0 +1,347 @@
+            for subset in test_subsets
+        ]
+
+    else:
+        subset_freq_hts = [
+            pre_process_subset_freq(subset, global_freq_ht) for subset in SUBSETS
+        ]
+
+    logger.info("Concatenating subset frequencies...")
+    freq_ht = hl.Table.multi_way_zip_join(
+        [global_freq_ht] + subset_freq_hts,
+        data_field_name="freq",
+        global_field_name="freq_meta",
+    )
+    freq_ht = freq_ht.transmute(freq=freq_ht.freq.flatmap(lambda x: x.freq))
+    freq_ht = freq_ht.transmute_globals(
+        freq_meta=freq_ht.freq_meta.flatmap(lambda x: x.freq_meta)
+    )
+
+    # Create frequency index dictionary on concatenated array (i.e., including all subsets)
+    # NOTE: non-standard downsampling values are created in the frequency script corresponding to population totals, so
+    # callset-specific DOWNSAMPLINGS must be used instead of the generic DOWNSAMPLING values
+    global_freq_ht = hl.read_table(get_freq().path)
+    freq_ht = freq_ht.annotate_globals(
+        freq_index_dict=make_freq_index_dict(
+            freq_meta=hl.eval(freq_ht.freq_meta),
+            pops=POPS,
+            downsamplings=hl.eval(global_freq_ht.downsamplings),
+        )
+    )
+
+    # Add back in all global frequency annotations not present in concatenated frequencies HT
+    row_fields = global_freq_ht.row_value.keys() - freq_ht.row_value.keys()
+    logger.info(
+        f"Adding back the following row annotations onto concatenated frequencies: {row_fields}"
+    )
+    freq_ht = freq_ht.annotate(**global_freq_ht[freq_ht.key].select(*row_fields))
+
+    global_fields = global_freq_ht.globals.keys() - freq_ht.globals.keys()
+    global_fields.remove("downsamplings")
+    logger.info(
+        f"Adding back the following global annotations onto concatenated frequencies: {global_fields}"
+    )
+    freq_ht = freq_ht.annotate_globals(
+        **global_freq_ht.index_globals().select(*global_fields)
+    )
+
+    logger.info("Preparing release Table annotations...")
+    ht = add_release_annotations(freq_ht)
+
+    logger.info("Removing chrM and sites without filter...")
+    ht = hl.filter_intervals(ht, [hl.parse_locus_interval("chrM")], keep=False)
+    ht = ht.filter(hl.is_defined(ht.filters))
+
+    ht = ht.checkpoint(
+        qc_temp_prefix() + "release/gnomad.genomes.v3.1.sites.chr20.ht"
+        if args.test
+        else release_sites().path,
+        args.overwrite,
+    )
+    logger.info(f"Final variant count: {ht.count()}")
+    ht.describe()
+    ht.show()
+    ht.summarize()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--test", help="Runs a test on chr20:1-1000000", action="store_true"
+    )
+    parser.add_argument(
+        "--test_subsets",
+        help="Specify subsets on which to run test, e.g. '--test_subsets non_v2 non_topmed'",
+        default=SUBSETS,
+        nargs="+",
+    )
+    parser.add_argument(
```

Comment:
@ch-kr Do you have insights into whether we should repartition the output sites Table? We're trying to decide whether to keep this argument

Reply:
I vote for removing this argument since we can now repartition on read directly through `hl.read_table`